setup.py

```diff
@@ -9,14 +9,14 @@
     author_email="tal@typesupply.com",
     url="https://github.com/unified-font-object/ufoNormalizer",
     package_dir={"": "src"},
-    py_modules=['ufonormalizer', '_version'],
+    packages={"ufonormalizer"},
```

Comment:
This works, but I'd have used a more general method

```python
from setuptools import setup, find_packages

setup(
   # ...
    package_dir={"": "src"},
    packages=find_packages(where="src"),
   # ...
)
```

Reply:
As above  -- will make this improvement next time. Thanks!